package.json

```diff
@@ -42,7 +42,7 @@
     "sdf-parser": "^7.0.4"
   },
   "peerDependencies": {
-    "openchemlib": ">=9.8.0"
+    "openchemlib": ">=9.12.0"
   },
```

Comment:
Note that this is technically a breaking change because it forces users of the library to update openchemlib. That's why the version was fixed in devDependencies.